libtmux/session.py

```diff
@@ -198,10 +198,10 @@ def rename_session(self, new_name: str) -> "Session":
     def new_window(
         self,
         window_name: t.Optional[str] = None,
-        start_directory: None = None,
+        start_directory: str = "",
         attach: bool = True,
         window_index: str = "",
-        window_shell: None = None,
+        window_shell: str = "",
```

Comment:
Can you explain these changes?

Reply:
Without fixing this, mypy complained about

```
tests/test_pane.py:72: error: Argument "window_shell" to "new_window" of "Session" has incompatible type "str"; expected "None"
```

If I had specified start_directory, the problem would have been the same, so fixed it at the same time

Reply:
@tony : what do you want me to do ?

Only fix window_shell typing with `t.Optional[str] = None` and fix the others later ?

Reply:
>  After re-reading the method and its docstring maybe I should have used t.Optional[str] as for window_name and update the docstring too. Let me know

Yes! This would be good

> What do you want me to do ?

Sorry for the delay

> Only fix `window_shell typing with `t.Optional[str] = None and fix the others later ?

I would prefer `t.Optional[str]`, the reason why is it'd preserve runtime behavior.

Reply:
> Sorry for the delay

NP ^^

Squashed commits
Rebased with master
Fixed correct type only for window_shell and add `optional` ind docstring